Documentation/contributor-guide/roadmap.md

```diff
@@ -0,0 +1,47 @@
+# roadmap
+
+etcd uses GitHub milestones to track all tasks in each major or minor release. The roadmap.md file only records the most
+important tasks for each release.
+Please also refer to [here](https://docs.google.com/spreadsheets/d/1hFpIbkI0IDum0SPy8txj7bVHxQRrAB_82f1Yqz_Pa8M/edit#gid=0&fvid=1685003197)
+for more context. Note that etcd will continue to focus on technical debt over the next few major or minor releases.
+
+## v3.6.0
```

Comment:
Can you provide clear definition of P0, P1, P2. Could be from https://github.com/etcd-io/etcd/blob/main/Documentation/postmortems/v3.5-data-inconsistency.md#action-items by just replacing v3.5 with the current milestone. My suggestion:
* P0 - Critical for for the current milestone. Blocks the release.
* P1 - Important for the current milestone, critical for the next milestone. 
* P2 - Nice to have, can be always skipped and should not block anything.

Reply:
Good call. Updated & thx